pkg/3scale/amp/component/apicast_monitoring.go

```diff
@@ -0,0 +1,189 @@
+package component
+
+import (
+	"fmt"
+
+	appsv1alpha1 "github.com/3scale/3scale-operator/pkg/apis/apps/v1alpha1"
+	"github.com/3scale/3scale-operator/pkg/assets"
+	"github.com/3scale/3scale-operator/pkg/common"
+	monitoringv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
+	grafanav1alpha1 "github.com/integr8ly/grafana-operator/v3/pkg/apis/integreatly/v1alpha1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func ApicastProductionMonitoringService() *v1.Service {
+	return &v1.Service{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Service",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "apicast-production-metrics",
+			Labels: map[string]string{
+				"app":                          appsv1alpha1.Default3scaleAppLabel,
+				"threescale_component":         "apicast",
+				"threescale_component_element": "production",
```

Comment:
Maybe we should take into account the PR #367 for this?

Reply:
Metering labels only apply to pods

Reply:
That's correct. I was referring to reusing the created options in that PR for "common" labels (not the metric ones)

Reply:
Ok, when #367 is merged, I will rebase this and apply changes. Leaving this conversation unresolved as reminder

Reply:
Done